<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>301</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="operations_widget">
       <property name="currentIndex">
        <number>5</number>
       </property>
       <widget class="QWidget" name="gray_scale_tab">
        <attribute name="title">
         <string>RGB to GS</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>221</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QSlider" name="red_slider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>33</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="green_slider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>33</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="blue_slider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>34</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLCDNumber" name="red_slider_display">
            <property name="value" stdset="0">
             <double>33.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLCDNumber" name="green_slider_display">
            <property name="value" stdset="0">
             <double>33.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLCDNumber" name="blue_slider_display">
            <property name="value" stdset="0">
             <double>34.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="brightness_change_tab">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <attribute name="title">
         <string>Brightness</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>221</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="brightness_changer_display"/>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="brightness_change_slider">
            <property name="minimum">
             <number>-255</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="negative_trashhold">
        <attribute name="title">
         <string>Negative with trashhold</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="negative_trashhold_display">
            <property name="value" stdset="0">
             <double>256.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="negative_trashhold_slider">
            <property name="maximum">
             <number>256</number>
            </property>
            <property name="value">
             <number>256</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="negative_trashhold_box">
            <property name="text">
             <string>On</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="binarization_tab">
        <attribute name="title">
         <string>Binarization</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="binarization_display">
            <property name="value" stdset="0">
             <double>256.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="binarizetion_slider">
            <property name="maximum">
             <number>256</number>
            </property>
            <property name="value">
             <number>256</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="binarization_box">
            <property name="text">
             <string>On</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="contrast_tab">
        <attribute name="title">
         <string>Contrast</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="contrast_q1_display"/>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="contrast_q1_slider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLCDNumber" name="contrast_q2_display">
            <property name="value" stdset="0">
             <double>255.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="contrast_q2_slider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="contrast_box">
            <property name="text">
             <string>On</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="contrast_decreas_mode_box">
            <property name="text">
             <string>Decreasing mode</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="gamma_tab">
        <attribute name="title">
         <string>Gamma</string>
        </attribute>
        <widget class="QCheckBox" name="gamma_box">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>On</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="gamma_box_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="quantization_tab">
        <attribute name="title">
         <string>Quantization</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="pseudo_coloring_tab">
        <attribute name="title">
         <string>Pseudo coloring</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="pseudo_q1_display"/>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="pseudo_q1_slider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLCDNumber" name="pseudo_q2_display"/>
          </item>
          <item row="2" column="0">
           <widget class="QLCDNumber" name="pseudo_q3_display"/>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="pseudo_q2_slider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="pseudo_q3_slider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="pseudo_box">
            <property name="text">
             <string>On</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Страница</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Страница</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>60</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="load_image_button">
           <property name="text">
            <string>Load Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_image_button">
           <property name="text">
            <string>Save Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="source_flag_box">
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="reset_button">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="preview_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hystogram_label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>440</y>
      <width>256</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
